avocado/core/messages.py

```diff
@@ -79,17 +79,16 @@ def process_message(self, message, task, job):
 
 class StartMessageHandler(BaseMessageHandler):
     """
-    Handler for started message.
+    Started message.
```

Comment:
If this is not a handler, then for consistency, we should change the class name too. Otherwise I think we should keep the handler here.

Reply:
I wasn't sure with this change. All of them are handlers, but this docstring, will be copied to the message documentation, so I thought that the word `Handler` might be confusing in documentation.

Reply:
I think that is fine to have "Handler" on the documentation if that makes sense.